src/components/assignment-panel/assignment-info/assignment-info.tsx

```diff
@@ -28,12 +29,12 @@ export const AssignmentInfo = ({  }: AssignmentInfoProps) => {
         let text = undefined
         let tooltip = undefined
         let filled = false
-        if (!assignment.isCreated) {
-            // Upcoming assignment (doesn't have either an open or close date yet)
+        if (!assignment.isCreated || !assignment.isAvailable) {
+            // Upcoming assignment (doesn't one either an open/close date OR open date not reached yet)
```

Comment:
This comment doesn't make sense, might want to reword it 😆

Reply:
fixed